_gsocproposals/2018/proposal_LHCbHEPDrone.md

```diff
@@ -3,7 +3,7 @@ title: Extension and characterisation of the LHCb data isolation drone neural ne
 layout: gsoc_proposal
 project: LHCb
 year: 2018
-organization: CERN, NWO, STFC
+organization: CERN, Nikhef, Edinburgh
```

Comment:
Please use the list syntax to specify more than one organisation, as explained here:
http://hepsoftwarefoundation.org/gsoc/guideline.html

Reply:
This has been changed, thanks